lib/libzfs/libzfs_pool.c

```diff
@@ -721,6 +722,17 @@ zpool_valid_proplist(libzfs_handle_t *hdl, const char *poolname,
 				goto error;
 			}
 			break;
+		case ZPOOL_PROP_DEDUPDITTO:
+			if (intval < ZIO_DEDUPDITTO_MIN && intval != 0) {
+				zfs_error_aux(hdl, dgettext(TEXT_DOMAIN,
+				    "property '%s' number %d is invalid, only "
+				    "values equal to 0 or higher than "
+				    "%" PRId32 " are allowed."),
+				    propname, intval, ZIO_DEDUPDITTO_MIN - 1);
+				(void) zfs_error(hdl, EZFS_BADVERSION, errbuf);
```

Comment:
I believe you meant `EZFS_BADPROP` here.

Reply:
Thanks, this is a copy/paste oversight.